### ISSUE TITLE
homebrew/science is been deprecated - manual on Seqan website still advises install with brew install homebrew/science/seqan 

### ISSUE THREAD

**Issue by aleighbrown** [seqan/seqan#2422]
Is it possible still to install seqan with homebrew? 

@rrahn It seems that homebrew/science is been deprecated, and that seqan (the library) migrated to homebrew core and that seqan-apps disappeared. Could you (or another mac user) investigate what the correct procedure now is?

_Originally posted by @h-2 in https://github.com/seqan/seqan/issues/2263#issuecomment-363105658_

**Comment by eseiler** [comment]
Hello,

can you try

```bash
brew install brewsci/bio/seqan@2
```

The package still exists and is up to date, but it was migrated.

**Comment by aleighbrown** [comment]
Install works -- leaving issue open as the docs are still incorrect

**Comment by aleighbrown** [comment]
Actually I ammend this;

Based on troubles I'm having reading BAM files, I think installing directly with homebrew does not correctly link against zlib.h

 https://github.com/seqan/seqan/issues/2056

**Comment by rrahn** [comment]
Good point with the documentation. We'll see that this will be updated soon. Regarding the installation of SeqAn. SeqAn is a header only library so there is nothing linked against. Furthermore ZLIB and BZIP are only optional and thus not required by the package install. In SeqAn3 we provide some example cmake script to start out working with SeqAn3. This would provide you then with proper diagnostics of what libraries have been found so it is easier to spot possible errors. I hope this helps a little.

**Comment by DennisSchwartz** [comment]
Quick ping to say that the documentation is still outdated :)

**Comment by SGSSGene** [comment]
Thank @DennisSchwartz for the reminder! and Thank you @eseiler  for the extrem fast fix!

**Comment by eseiler** [comment]
The only drawback is that we can't edit the files on seqan.readthedocs.io, which are hosted on ReadTheDocs' CDN.
E.g., we can't make the 2.5.1 docs also show the updated packages page.
But it's at least up-to-date for the "main" branch, which is also the default and the one we link to.
